codalab/apps/web/static/js/worksheet/table_item_interface.jsx

```diff
@@ -112,6 +113,20 @@ var TableRow = React.createClass({
         this.props.updateRowIndex(this.props.rowIndex);
     },
 
+    handleOptionButtonClick(event) {
+        event.nativeEvent.stopImmediatePropagation();
```

Comment:
Out of curiosity -- could you explain to me why `stopImmediatePropagation` is used rather than `preventDefault`?